Clojure/tests/syntax_test_clojurescript_old.cljs

```diff
@@ -0,0 +1,103 @@
+; SYNTAX TEST "Packages/Clojure/ClojureScript.sublime-syntax"
```

Comment:
Since the syntax just imports the main clojure syntax, I don't see a need to duplicate the entire test suite. It would also need to be kept in sync.

Reply:
Sound good.